packages/api/tsconfig.dist.json

```diff
@@ -11,5 +11,5 @@
     "outDir": "dist"
   },
   "include": ["."],
-  "exclude": ["dist"]
+  "exclude": ["dist", "**/*.test.js"]
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏ideally we shouldn't exclude test files. Also: the test files should be `.ts` not `.js`.

Reply:
It looks like typecheck still runs. But I still took it out as it wasn't needed after the other changes you requested, and the other packages aren't excluding the tests at this time.